Makefile

```diff
@@ -1,5 +1,6 @@
 # Go parameters
 GOCMD?=go
+GO_VERSION=$(subst go,,$(word 3, $(shell go version)))  # parses "go version go1.15 darwin/amd64"
```

Comment:
I figured the current go version might be safest to key off

Reply:
I think it's better to hardcode so that different users don't get different results based on the state of their machines.

If we want to be clever and avoid hardcoding in two places we could grab it from the [mod file](https://github.com/buildpacks/imgutil/blob/main/go.mod#L16) with `go list -m -f "{{.GoVersion}}"`.

Reply:
Thanks for that command. I was struggling to figure out how to do exactly that!